tools/celtest/testdata/raw_expr_tests.yaml

```diff
@@ -32,3 +32,19 @@ section:
             value: 42
         output:
           value: false
+  - name: "unknown"
```

Comment:
for completeness, could you add a case where multiple expr ids are returned as unknown?